src/utils/tag-object.ts

```diff
@@ -1,15 +1,13 @@
 const redactableKeys = ["authorization", "x-authorization", "password", "token"];
-const maxDepth = 10;
 
-export function tagObject(currentSpan: any, key: string, obj: any, depth = 0): any {
+export function tagObject(currentSpan: any, key: string, obj: any, depth = 0, maxDepth = 10): any {
   if (depth >= maxDepth) {
-    return;
-  } else {
-    depth += 1;
+    return currentSpan.setTag(key, redactVal(key, JSON.stringify(obj).substring(0, 5000)));
```

Comment:
So the big change is now we fall back to tagging the entire `rest` of the object as one string, but otherwise the rest is the same, right?

Reply:
that is correct.